docs-src/getting-started.md

```diff
@@ -0,0 +1,55 @@
+# Getting started with Services APIs
+
+## Installing CRDs
+
+Service APIs is a collection of Kubernetes Custom Resource Definitions that need
+to be installed into any Kubernetes cluster.
+
+To install the CRDs, please execute:
+
+```
+kubectl kustomize "github.com/kubernetes-sigs/service-apis/config/crd?ref=master" \
+| kubectl apply -f -
+```
+
+## Install an implementation
+
+Service APIs is a set of specification that multiple implementations confirm to.
```

Comment:
This doesn't feel quite right. Maybe "Multiple projects implement the APIs defined by this project."